### PR TITLE
compile with and against api 23

### PR DESCRIPTION
**Pull request by larsgrefer**

### CODE REVIEW

**Comment by kattrali** [comment]
:+1: :star: Thanks!

File: gradle.properties [hunk 1]
```diff
@@ -16,6 +16,6 @@ POM_URL=https://bugsnag.com
 POM_ARTIFACT_ID=bugsnag-android
 POM_PACKAGING=jar
 
-ANDROID_BUILD_TARGET_SDK_VERSION=19
-ANDROID_BUILD_TOOLS_VERSION=21.1.1
-ANDROID_BUILD_SDK_VERSION=19
```
Comment (larsgrefer) [inline gradle.properties]: Just because im curious: Why are these 3 values defined here and not in the build.gradle?

/cc @loopj

Comment (loopj) [inline gradle.properties]: ¯\_(ツ)_/¯ - I can't remember the reasoning behind this :)

Comment (larsgrefer) [inline gradle.properties]: I've never seen this before and can't think of any advantages

In fact it has the disadvantage that AndroidStudio can't interprete the values and can't show hints about newer versions

**Pull request merged**